walker/src/components/AuthWrapper.tsx

```diff
@@ -0,0 +1,42 @@
+import { useEffect } from "react";
+import { Outlet, useNavigate } from "react-router-dom";
+
+function AuthWrapper() {
+    const navigate = useNavigate()
+    const token = sessionStorage.getItem("token");
+    const BACKEND_URL = process.env.REACT_APP_PROD === "true" ? process.env.REACT_APP_BACKEND_URL_PROD : process.env.REACT_APP_BACKEND_URL_DEV
```

Comment:
nit: could we make BACKEND_URL a shared variable (in a file like constants.ts) & just import since we use it multiple times

also might be simpler to just have REACT_APP_BACKEND_URL_PROD as a variable only in production and just do `const BACKEND_URL = process.env.REACT_APP_BACKEND_URL_PROD ?? process.env.REACT_APP_BACKEND_URL_DEV`